CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `version` subcommand to show the actual build version
   ([Issue #24](https://github.com/cycloidio/terracognita/issues/24))
 
+### Changed
+
+- The Terraform version from 0.11 to 0.12 with all the implications (file formats)
+  ([PR #33](https://github.com/cycloidio/terracognita/pull/33))
```

Comment:
Could make sense to drop a link to https://www.terraform.io/upgrade-guides/0-12.html